_overviews/scala3-book/taste-indentation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Indentation
+type: section
+description: This section demonstrates Scala 3 Indentation.
```

Comment:
I don’t think it’s correct to say that we “demonstrate” indentation. What you could say, instead is that this section shows that some lexical scopes can be delimited by indentation instead of braces.